src/master-playlist-controller.js

```diff
@@ -711,24 +711,38 @@ export class MasterPlaylistController extends videojs.EventTarget {
       return false;
     }
 
-    // when the video is a live stream
-    if (!media.endList) {
+    // when the video is a live stream, or has a specific start time:
+    if (!media.endList || media.start) {
       const seekable = this.seekable();
 
       if (!seekable.length) {
-        // without a seekable range, the player cannot seek to begin buffering at the live
-        // point
+        // without a seekable range, the player cannot seek to a specific time
         return false;
       }
 
+      const seekableTime = seekable.end(0);
+      let startTime = seekableTime;
+
+      if (media.start) {
+        const offset = media.start.timeOffset;
+
+        if (offset < 0) {
+          startTime += offset;
```

Comment:
I think `seekable.end(0)` should be the "safe seekable end" meaning 3 segments from the end of the playlist, whereas the spec says the offset should be from the end of the playlist.

Would be good to add a test for that.